### PR TITLE
backend of /api/users.csv with tests #2281

### PR DESCRIPTION
**Pull request by awick**
Still needs @31453 UI magic to surface

### CODE REVIEW

File: common/arkimeUtil.js [hunk 1]
```diff
@@ -71,6 +71,15 @@ class ArkimeUtil {
     return next();
   }
 
+  // ----------------------------------------------------------------------------
+  static noCache (req, res, ct) {
```
Comment (31453) [inline common/arkimeUtil.js:75]: Isn't this already in ViewerUtils? We should remove it from there and use this everywhere

**Comment by 31453** [comment]
The UI for this would just be a button on the Users page to download the csv (passing in the query params)?

File: common/user.js [hunk 2]
```diff
@@ -510,6 +510,69 @@ class User {
...
+    if (Array.isArray(req.body.start) || Array.isArray(req.body.length)) {
+      return res.send('Invalid');
+    }
+
+    const query = {
+      from: parseInt(req.body.start) || 0,
+      size: parseInt(req.body.length) || 10000
+    };
+
+    if (ArkimeUtil.isString(req.body.filter)) {
```
Comment (31453) [inline common/user.js:532]: We do this in `apiGetUsers` too. Should we abstract this to a new function?

File: common/arkimeUtil.js (hunk 1 above)
Comment (awick) [reply common/arkimeUtil.js:75]: yep, was going to do another PR for that

**Comment by awick** [comment]
> The UI for this would just be a button on the Users page to download the csv (passing in the query params)?

yep, whenever you have time

File: common/user.js (hunk 2 above)
Comment (awick) [reply common/user.js:532]: ok, will do

**Comment by awick** [comment]
LGTM

**Pull request merged**